src/templates/admin/review/home.html

```diff
@@ -50,8 +50,18 @@ <h2>Articles in Review</h2>
                                     {% for review in article.reviewassignment_set.all %}
 
                                         <tr>
-                                            <td><a href="{% url 'review_view_review' article.pk review.pk %}">{{ review.reviewer.initials }}</a></td>
-                                            <td><span class="{{ review.status.span_class }}">{{ review.status.display|safe }}</span></td>
+                                            <td>({{ review.review_round.round_number }}) <a href="{% url 'review_view_review' article.pk review.pk %}">{{ review.reviewer.initials }}</a></td>
+                                            <td>
+                                                <span class="{{ review.status.span_class }}">
+                                                    {% if review.status.code == 'complete' %}
+                                                    <span data-tooltip class="top" tabindex="2" title="Recommendation: {{ review.get_decision_display }}">
+                                                    {% endif %}
+                                                    {{ review.status.display|safe }}
+                                                    {% if review.status.code == 'complete' %}
+                                                    </span>
+                                                    {% endif %}
```

Comment:
As discussed:
- Ignore reviews with status 'withdrawn'
- Instead of rendering the round, only show review assignments relevant to this round
Can we also add a different message when the article is under revisions (There exists an open RevisionRequest)

Reply:
@mauromsl I've updated this in consultation with @rhb123. Decision was made to continue to show all rounds though.